components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -2056,9 +2060,18 @@ def getAllTypes(descriptors, dictionaries, callbacks):
     for callback in callbacks:
         for t in getTypesFromCallback(callback):
             yield (t, None, None)
+    for typedef in typedefs:
+        yield (typedef.innerType, None, None)
+
+
+def hasTypedefVariant(t):
+    for memberType in t.memberTypes:
+        if isinstance(memberType, IDLTypedefType):
+            return True
+    return False
```

Comment:
This function here is pointless because none of the types passed in ever reaches the true branch condition.